tests_new/integration_tests/modules/datasets_base/test_dataset.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from assertpy import assert_that
+
+from integration_tests.modules.datasets_base.queries import (
+    list_datasets,
+    list_owned_datasets,
+    list_datasets_created_in_environment,
+)
+
+log = logging.getLogger(__name__)
+
+
+def test_list_datasets(
+    client1, session_s3_dataset1, session_imported_sse_s3_dataset1, session_imported_kms_s3_dataset1, session_id
+):
+    assert_that(list_datasets(client1, term=session_id).nodes).is_length(3)
```

Comment:
In case further we have to add new datasets, may be it's better not to fix the number?
Anyway, it's better to check the datasets by uri, I think

Reply:
Because it is a list, I wanted to check that the listing of items returns the expected items. But it is true that it can cause issues with other existing test resources. That's why the `term=session_id` filter is applied

Reply:
I already has this problem with my share_tests, but in table tests. I created another table for sharing tests, so e.g. here I have 3 tables, not 2
```
def test_list_dataset_tables(client1, dataset_fixture_name, tables_fixture_name, request):
    tables = request.getfixturevalue(tables_fixture_name)
    dataset = request.getfixturevalue(dataset_fixture_name)
    response = list_dataset_tables(client1, dataset.datasetUri)
    assert_that(response.tables.count).is_equal_to(2)
```

Reply:
May be I should reuse the existing datasets and don't create others, but I'll revise it after this PR is merged.
Anyway, in future we may have to add more tables/folders/datasets

Reply:
I understood that dataset test objects are controlled inside the dataset tests. Other modules can use the fixtures defined in global_conftest. So yes, I think in shares you should be using the datasets/tables and folders from the datasets module